cli/util/api.go

```diff
@@ -119,6 +120,15 @@ func GetAPIInfo(ctx *cli.Context, t repo.RepoType) (APIInfo, error) {
 		return APIInfo{}, xerrors.Errorf("could not open repo at path: %s; %w", p, err)
 	}
 
+	exists, err := r.Exists()
+	if err != nil {
+		return APIInfo{}, xerrors.Errorf("repo.Exists returned an error: %w", err)
+	}
+
+	if !exists {
+		return APIInfo{}, errors.New("repo directory does not exist. Make sure your configuration is correct")
```

Comment:
What do we print if you run `lotus-miner init ..` without a full-node repo? (just want to make sure we won't make that message really confusing)

Reply:
```
# repo dir doesn't exist
➜  LOTUS_PATH=~/.lotussss LOTUS_MINER_PATH=~/.restoredrepo ./lotus-miner init service --type=markets --api-sealer=$APISEALER --api-sector-index=$APISEALER --config=~/.lotusmarket/config.toml ~/.lotusbackup/backupfile
2021-07-29T14:54:08.146+0300    INFO    main    lotus-miner/init_service.go:54  Initializing lotus miner service
2021-07-29T14:54:08.147+0300    INFO    main    lotus-miner/init_restore.go:102 Trying to connect to full node RPC
ERROR: could not get API info: repo directory does not exist. Make sure your configuration is correct

# no LOTUS_PATH, so ~/.lotus is inferred and it just works
LOTUS_MINER_PATH=~/.restoredrepo ./lotus-miner init service --type=markets --api-sealer=$APISEALER --api-sector-index=$APISEALER --config=~/.lotusmarket/config.toml ~/.lotusbackup/backupfile
```